### ISSUE TITLE
training loss increases when finetuning on master64 on VCTK

### ISSUE THREAD

**Issue by pranaymanocha** [facebookresearch/denoiser#12]
Hi!
Thanks for the great repo. The work is awesome!
When I tried to finetune a denoising network on VCTK dataset, I observe that the train loss increases after a few epochs. I am loading the pretrained model "master64" and then finetuning on the VCTK dataset. Here is my forked repo with more details:[link](https://github.com/pranaymanocha/denoiser). Please look at the "history.json" to check the training history.

Thanks!

**Comment by adiyoss** [comment]
Hi,
@pranaymanocha, from what I know VCTK is a pretty clean dataset, did you inject any noise to it?

**Comment by pranaymanocha** [comment]
Hi,
I think the loss increases because of my low batch_size (2) with the same learning rate (3e-4). I think that's what causing the loss to increase, though I don't fully understand why the training loss increases. I could update this later as I find out more, but I'll close it for now. Thanks!
